src/frontend/styles/components/sidebar.pcss

```diff
@@ -73,11 +73,46 @@
     }
   }
 
+  &__search {
+    @apply --input;
+
+    appearance: none;
+    background: url("../svg/search.svg") left 10px center no-repeat;
+    background-color: var(--color-input-primary);
+    border: 1px solid transparent;
+    border-radius: 8px;
+    padding: 7.5px 35px;
+    padding-right: 42px;
+    line-height: 17px;
+    font-size: 14px;
+
+    &-wrapper::after {
+      color: var(--color-button-secondary);
+      font-weight: 600;
```

Comment:
It is not bold in Figma

Stage:
<img width="362" alt="image" src="https://user-images.githubusercontent.com/3684889/195421967-8b2fe578-ce52-49f1-af50-61afbe937cb5.png">

Figma:
<img width="328" alt="image" src="https://user-images.githubusercontent.com/3684889/195422495-63fd6f99-1b2e-4923-bb90-7b20692ec8c3.png">

Reply:
done